docs/source/reference/envvars.rst

```diff
@@ -467,6 +467,12 @@ GPU support
    Strict strides checking is deprecated and may be removed in future. See
    :ref:`deprecation-strict-strides`.
 
+.. envvar:: NUMBA_CUDA_LOW_OCCUPANCY_WARNINGS
+
+   Enable warnings if the block size is too small relative to the number of
```

Comment:
The error is generated when the grid size is too small, not the block size. It's probably worth mentioning that the heuristic used here is `(grid size) < 2 * (number of SMs)`, and that the absence of the warning doesn't necessarily imply a good grid size - larger multiples of the SM count may be more efficient.